ci-operator/step-registry/workers-scale/workers-scale-commands.sh

```diff
@@ -0,0 +1,93 @@
+#!/bin/bash
+
+log(){
+    echo -e "\033[1m$(date "+%d-%m-%YT%H:%M:%S") " "${*}\033[0m"
+}
+
+function waitForReady() {
+    echo "Wait for Ready"
+    set +x
+    local retries=0
+    local attempts=140
+    while [[ $(oc get nodes --no-headers -l node-role.kubernetes.io/worker | grep -v "NotReady\\|SchedulingDisabled" | grep worker -c) != "$1" ]]; do
```

Comment:
@paigerube14 This is not getting current worker count so your change https://github.com/openshift/release/pull/39269 won't effect , but may be we can use similar command pointed by Joe here as well?

Reply:
I will have the https://github.com/openshift/release/pull/39269 edit the command to get the number of nodes without grep since this is passing already here, will fine toon in later pr

Reply:
The above commands are not addressed as part of https://github.com/openshift/release/pull/39269 but Paige or I will make sure to edit the commands in next PR.